gcc/Makefile

```diff
@@ -6,7 +6,7 @@ ifeq ($(GCC),)
 GCC := gcc
 endif
 
-CFLAGS=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -ggdb -fPIC -DIRIG_NETWORKING -DSHA256ENABLE -Wall
+CFLAGS=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -ggdb -fPIC -DIRIG_NETWORKING -DSHA256ENABLE -Wall -Wno-address-of-packed-member
```

Comment:
We accept that we will systematically be adressing structures without proper alignment. Most compilers will handle this behavior correctly.